docs/dotnet/dotnet-kestrel-metrics.md

```diff
@@ -138,6 +138,50 @@ different processes could be listening on TCP port 12345 and UDP port 12345.
 
 | Value  | Description | Stability |
 |---|---|---|
+| `aborted_by_app` | The HTTP/1.1 connection was aborted when app code aborted an HTTP request with `HttpContext.Abort()`. | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| `app_shutdown_timeout` | The connection was aborted during app shutdown. During shutdown, the server stops accepting new connections and HTTP requests, and it is given time for active requests to complete. If the app shutdown timeout is exceeded, all remaining connections are aborted. | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| `closed_critical_stream` | A critical control stream for an HTTP/3 connection was closed. | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| `connection_reset` | The connection was reset while there were active HTTP/2 or HTTP/3 streams on the connection. | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| `error_after_starting_response` | An error occurred after the response was started, causing an abort of the HTTP/1.1 connection or multiplexed request stream. | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
```

Comment:
What does "multiplexed request stream" mean here? I would have suspected it's same thing as "HTTP/2 or HTTP/3 stream", but I don't see why we wouldn't just abort the individual stream rather than the entire connection in that case. Reading the code, it looks like `error_after_starting_response` connection end reason only gets set via the `Http1Connection.DisableKeepAlive` override.

```suggestion
| `error_after_starting_response` | An error such as an unhandled application exception or invalid request body occurred after the response was started, causing an abort of the HTTP/1.1 connection. | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
```